helm/thingsboard/values.yaml

```diff
@@ -394,9 +394,9 @@ sql:
     batch:
       threads: 10
     ttl:
-      enabled: false
-      value: 86400000
-      execution_interval: 0
+      enabled: true
+      value: 0
```

Comment:
are the values right? 
what does value=0 mean?

Reply:
No idea, but they are the default ones in ThingsBoard